apps/fluent-tester/src/FluentTester/TestComponents/RadioGroup/RadioGroupTest.tsx

```diff
@@ -47,20 +48,47 @@ const basicRadioGroup: React.FunctionComponent = () => {
   );
 };
 
-const radioGroupSections: TestSection[] = [
-  {
+const nativeRadioGroup: React.FunctionComponent<{}> = () => {
+  return (
+    <View>
+      <RadioGroup label="RadioGroup 1">
+        <NativeRadioButton title="Option A" state={true} onPress={() => alert('Option A pressed')} buttonKey="A" />
+        <NativeRadioButton title="Option B" buttonKey="B" />
+        <NativeRadioButton title="Option C (disabled)" enabled={false} style={{ width: 150 }} buttonKey="C" />
+        <NativeRadioButton title="Option D" buttonKey="D" />
+      </RadioGroup>
+      <Separator />
+      <RadioGroup label="RadioGroup 2">
+        <NativeRadioButton title="Option A" buttonKey="A" />
+        <NativeRadioButton title="Option B" state={true} onPress={() => alert('Option B pressed')} buttonKey="B" />
+        <NativeRadioButton title="Option C (disabled)" enabled={false} style={{ width: 150 }} buttonKey="C" />
+        <NativeRadioButton title="Option D" buttonKey="D" />
+      </RadioGroup>
```

Comment:
I think we should rename the `state` prop to somethingg like `isSelected`. state feels very misleading since that's something very special in React terminology.

Reply:
fixed in commit fb736cb070687bb15f4028a3bafafaeccc538eec